webrtc/interfaces.html

```diff
@@ -0,0 +1,118 @@
+<!doctype html>
+<html>
+  <head>
+    <meta charset=utf-8>
+    <title>IDL check of RTCPeerConnection</title>
+    <link rel="author" title="Soares Chen" href="mailto:soares.chen@gmail.com"/>
+    <link rel="help" href="http://w3c.github.io/webrtc-pc/">
+  </head>
+  <body>
+    <h1 class="instructions">Description</h1>
+    <p class="instructions">This test verifies the conformance of WebIDLs defined in WebRTC</p>
+    <div id="log"></div>
+    <script src=/resources/testharness.js></script>
+    <script src=/resources/testharnessreport.js></script>
+    <script src=/resources/WebIDLParser.js></script>
+    <script src=/resources/idlharness.js></script>
+    <script>
+     (() => {
+       'use strict'
+
+       function initRTCPeerConnection(idlArray) {
+         window.pc = new RTCPeerConnection(null);
+         idlArray.add_objects({'RTCPeerConnection': ['pc']});
+       }
+
+       function initRTCIceCandidate(idlArray) {
+         window.iceCandidate = new RTCIceCandidate({
+           candidate: 'candidate:1467250027 1 udp 2122260223 192.168.0.196 46243 typ host generation 0'
+         });
+         idlArray.add_objects({'RTCIceCandidate': ['iceCandidate']});
+       }
+
+       function initRTCSessionDescription(idlArray) {
+         window.sessionDesc = new RTCSessionDescription({ type: 'offer' });
+         idlArray.add_objects({'RTCSessionDescription': ['sessionDesc']});
+       }
+
+       function initRTCIceEvent(idlArray) {
+         window.iceEvent = new RTCPeerConnectionIceEvent('ice-event', { });
```

Comment:
Ah, that's wrong, good catch. Be sure to always look at the normative bits (mostly IDL, algorithms, and things that say "must") and not non-normative description. If it's ever not clear in a spec if a part is supposed to be the requirement or the explanation of it, please file spec issues :)